### PR TITLE
Nisha | Test | Sales & purchase in material table sticky

### PR DESCRIPTION
**Pull request by nisha-walkover**
* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior (if this is a feature change)?**

* **Other information**:

### CODE REVIEW

File: apps/web-giddh/src/assets/locale/reports/purchase-register/en.json [hunk 1]
```diff
@@ -11,7 +11,8 @@
     "net_tcs": "Net TCS",
     "voucher_type": "Voucher Type",
     "voucher_no": "Voucher No.",
-    "qty_rate": "Qty-Rate",
+    "qty_rate": "Rate",
```
Comment (dvCodeWorld) [inline apps/web-giddh/src/assets/locale/reports/purchase-register/en.json]: use common locale json

app_unit
app_rate

File: apps/web-giddh/src/app/reports/components/purchase-register-expand-component/purchase.register.expand.component.html [hunk 2]
```diff
@@ -46,269 +46,293 @@
...
-                        [hidden]="
-                            !(purchaseRegisteDetailedResponse$ | async)?.items?.length && !voucherNumberInput?.value
-                        "
-                    >
-                        <tr>
-                            <th *ngIf="showFieldFilter.date">
-                                <div class="d-flex">
+        <div class="clearfix sales-register-table mb-5 overflow-visible">
+            <div class="responsive table-responsive overflow-visible table-header-sticky">
+                <ng-container *ngIf="(purchaseRegisteDetailedResponse$ | async)?.items?.length > 0">
```
Comment (dvCodeWorld) [inline apps/web-giddh/src/app/reports/components/purchase-register-expand-component/purchase.register.expand.component.html]:
```suggestion
                <ng-container *ngIf="(purchaseRegisteDetailedResponse$ | async)?.items?.length">
```

File: apps/web-giddh/src/app/reports/components/sales-register-expand-component/sales.register.expand.component.html [hunk 3]
```diff
@@ -247,72 +247,104 @@ <h5>
...
-                <table class="table basic giddh-table on-mobile-view">
-                    <thead
-                        [hidden]="!(salesRegisteDetailedResponse$ | async)?.items?.length && !voucherNumberInput?.value"
-                    >
-                        <tr>
-                            <th *ngIf="showFieldFilter.date">
-                                <div class="d-flex">
+        <div class="sales-register-table mb-5 overflow-visible">
+            <div class="responsive table-responsive overflow-visible table-header-sticky">
+                <ng-container *ngIf="(salesRegisteDetailedResponse$ | async)?.items?.length > 0">
```
Comment (dvCodeWorld) [inline apps/web-giddh/src/app/reports/components/sales-register-expand-component/sales.register.expand.component.html]:
```suggestion
                <ng-container *ngIf="(salesRegisteDetailedResponse$ | async)?.items?.length">
```

**Review by dvCodeWorld — CHANGES_REQUESTED** [review]

File: apps/web-giddh/src/app/reports/components/sales-register-expand-component/sales.register.expand.component.html (hunk 3 above)
Comment (nisha-walkover) [reply apps/web-giddh/src/app/reports/components/sales-register-expand-component/sales.register.expand.component.html]: done

File: apps/web-giddh/src/app/reports/components/purchase-register-expand-component/purchase.register.expand.component.html (hunk 2 above)
Comment (nisha-walkover) [reply apps/web-giddh/src/app/reports/components/purchase-register-expand-component/purchase.register.expand.component.html]: done

File: apps/web-giddh/src/assets/locale/reports/purchase-register/en.json (hunk 1 above)
Comment (nisha-walkover) [reply apps/web-giddh/src/assets/locale/reports/purchase-register/en.json]: done

**Review by dvCodeWorld — APPROVED** [review]

**Pull request merged**